web/src/app/core/services/repository.service.ts

```diff
@@ -51,4 +51,27 @@ export class RepositoryService {
     const callable: any = this.fns.httpsCallable('deleteGitWebhookRepository');
     return callable({ repositoryUid, token: this.authService.profile.oauth.githubToken });
   }
+
+  public getRepoRating(repo: RepositoryModel): number {
```

Comment:
Because already in `repository` service, `getRating` should be descriptive enough, as it would read `respositoryService.getRating(...)`

Reply:
Sure !!